tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -81,13 +81,14 @@ export const inviteMembers = async (args: inviteMembersArgs): Promise<void> => {
   const { page, role, recipients } = args
   for (const recipient of recipients) {
     const shareInputLocator = page.locator(invitationInput)
+    await shareInputLocator.click()
     await Promise.all([
       page.waitForResponse((resp) => resp.url().includes('sharees') && resp.status() === 200),
       shareInputLocator.fill(recipient.id)
     ])
     await shareInputLocator.focus()
     await page.waitForSelector('.vs--open')
-    await page.locator(invitationInput).press('Enter')
+    await page.locator('.vs__dropdown-option').click()
```

Comment:
I don't know why these changes are necessary, but without them, e2e tests won't succeed. I assume this happens because the panel is now loading way faster than before... but that's just a guess.